test/runtests.jl

```diff
@@ -15,4 +15,9 @@ end
     @test TestSetup.getfloat() isa Float64
 end
 
+dir = pwd()
```

Comment:
I am not sure if there is any point testing this. This also passes with global `cd`